components/ButtonGenerator/CodeBlock.tsx

```diff
@@ -70,6 +70,51 @@ export default function CodeBlock({ button }): JSX.Element {
     }
   };
 
+  const checkCurrency = (value, type) => {
```

Comment:
Feels like we could come up with a better function name here but I don't have any suggestions.